agent_build_refactored/tools/run_in_ec2/cleanup_ec2_objects.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2014-2022 Scalyr Inc.
```

Comment:
The same. Just moved `tests/end_to_end_tests/run_in_remote_machine/cleanup_test_ec2_objects.py` And replaced libcloud  with boto3